tests/core/sinkhorn_lr_test.py

```diff
@@ -69,7 +69,7 @@ def test_euclidean_point_cloud_lr(
         initializer=initializer,
     )
     solved = solver(ot_prob)
-    costs = solved.costs
+    costs = solved.errors
```

Comment:
sorry, I must be missing something, but wasn't the idea here to monitor the cost of the OT solution, not the convergence criterion? and then to compare it below?

Reply:
You're right, I wanted to rename `criterions -> errors`, not `costs`. Fixed.